noxfile.py

```diff
@@ -53,7 +53,7 @@ def test(session):
 
 @nox.session()
 def format(session):
-    session.install("black", "isort", "flynt", "unasync")
+    session.install("black", "isort", "flynt", "unasync", "setuptools")
```

Comment:
setuptools is no longer installed by default in recent versions of Python, but unasync requires it.